ui/component/or-asset-tree/src/index.ts

```diff
@@ -269,6 +271,13 @@ export class OrAssetTree extends subscribe(manager)(LitElement) {
     protected _expandedNodes: UiAssetTreeNode[] = [];
     protected _initCallback?: EventCallback;
 
+    @property({attribute: false})
+    protected _filterValue: string = '';
+    @property({type: Boolean})
+    protected _isFiltering: boolean = false;
+    protected _foundAssets: string[] = [];
+    protected _searchInputTimer: ReturnType<typeof setTimeout> | undefined = undefined;
```

Comment:
`protected _searchInputTimer?;` saves having to explicitly state `| undefined` also happy to have type as `number`

Reply:
I migrate to standard number